lib/id_helper.ex

```diff
@@ -1,57 +0,0 @@
-defmodule SnowflexId do
-  @moduledoc """
-  Here is where the ID is actually calculated using custom or
-  elixir epoch (since first commit).
-  """
-  @spec node_limit :: integer
-  def node_limit, do: 1023
-
-  @spec sequence_limit :: integer
-  def sequence_limit, do: 4095
-
-  @elixir_epoch :erlang.universaltime_to_posixtime({{2011, 1, 9}, {9, 46, 8}}) * 1000
-  @custom_epoch Application.get_env(:snowflexid, :epoch, @elixir_epoch)
-
```

Comment:
The removal of this attribute was an important change. You shouldnt use Application.get_env/3 in module attributes when writing libraries because they require a recompilation of the dependency (which isnt triggered by config changes)

Reply:
True, I remember a problem with release regarding module attributes because they are compile-time.
However let's keep the configuration in runtime so that the library can be used with legacy data.

Reply:
I was using module attribute because this simple change would cause this:

Comparison (with custom_epoch/0 function): 
snowflexid_sequence        138.42
snowflexid_table            93.89 - 1.47x slower +3.43 ms
ecto_uuid                   19.81 - 6.99x slower +43.26 ms
Comparison (with @custom_epoch): 
snowflexid_sequence        280.09
snowflexid_table           119.68 - 2.34x slower +4.79 ms
ecto_uuid                   21.42 - 13.07x slower +43.11 ms